Makefile

```diff
@@ -16,5 +16,10 @@ clean-docs:
 	find ./docs/docs -name "*.ipynb" -type f -delete
 	rm -rf docs/site
 
+format format_diff:
+	poetry run ruff format libs/ examples/
```

Comment:
we have individual formatters in each library, so probably let's remove `libs` from here

Reply:
Hm ok